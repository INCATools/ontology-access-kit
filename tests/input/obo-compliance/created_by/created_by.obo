format-version: 1.4
ontology: created_by

[Term]
id: X:1
created_by: username

