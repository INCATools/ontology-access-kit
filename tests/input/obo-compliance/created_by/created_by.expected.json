{
  "graphs" : [ {
    "id" : "http://purl.obolibrary.org/obo/created_by.owl",
    "meta" : {
      "basicPropertyValues" : [ {
        "pred" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
        "val" : "1.4"
      } ]
    },
    "nodes" : [ {
      "id" : "http://purl.obolibrary.org/obo/X_1",
      "type" : "CLASS",
      "meta" : {
        "basicPropertyValues" : [ {
          "pred" : "http://www.geneontology.org/formats/oboInOwl#created_by",
          "val" : "username"
        } ]
      }
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
      "lbl" : "has_obo_format_version",
      "type" : "PROPERTY"
    } ]
  } ]
}